src/libraries/Common/src/System/AppContextSwitchHelper.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+
+namespace System
+{
+    internal static class AppContextSwitchHelper
```

Comment:
I'm surprised something like this doesn't exist yet.
BTW, we could use this in other networking places like sockets and HTTP, where have some switches as well and apparently we have the same helpers copy-pasted there as well...

Reply:
> I'm surprised something like this doesn't exist yet.

There is a similar helper in coreclrlib system.dll, but The code can't be shared because it calls internal methods on `bool`.

I was also surprised, and have on my list to follow this up later to unify the code, at least in networking.